src/Admin.php

```diff
@@ -32,6 +32,9 @@ public static function init() {
     // Exposes SVG images in media library.
     add_filter('wp_prepare_attachment_for_js', __CLASS__ . '::wp_prepare_attachment_for_js');
     add_action('admin_head', __CLASS__ . '::admin_head');
+
+    // Stops MCE editor setting unwanted rel attribute values for links that open in new window.
+    add_filter('tiny_mce_before_init', __CLASS__ . '::tinymce_allow_unsafe_link_target');
```

Comment:
The method name should be equal to the hook name, unless there is a reason to use a different name.